ADALiOS/ADALiOS/NSString+ADExtentions.m

```diff
@@ -0,0 +1,45 @@
+//
+//  NSString+ADExtentions.m
+//  ADALiOS
+//
+//  Created by Brandon Werner on 9/11/15.
+//  Copyright © 2015 MS Open Tech. All rights reserved.
+//
+
+#import "NSString+ADExtentions.h"
+
+@implementation NSString (ADExtensions)
+
+- (NSString*)adSpaceDeliminatedString
```

Comment:
Yes, but you need this in various places in the code and it wasn't there. Makes sense to follow the pattern and add a category. It's up to you - but I think for the OSS developer seeing NSString equally amongst NSSet and NSArray categories for ADExtensions implementing `adSpaceDelimitedString` will help them understand what the code is doing even if we can actually make it fancy.